composer.json

```diff
@@ -41,56 +42,13 @@
     "lint:embed-optimizer": "@lint -- ./plugins/embed-optimizer --standard=./plugins/embed-optimizer/phpcs.xml.dist",
     "lint:optimization-detective": "@lint -- ./plugins/optimization-detective --standard=./plugins/optimization-detective/phpcs.xml.dist",
     "lint:speculation-rules": "@lint -- ./plugins/speculation-rules --standard=./plugins/speculation-rules/phpcs.xml.dist",
-    "lint:webp-uploads": "@lint -- ./plugins/webp-uploads --standard=./plugins/webp-uploads/phpcs.xml.dist",
-    "test": "phpunit --verbose --testsuite performance-lab",
```

Comment:
Why remove all of the test scripts from composer? I think it would be better if scripts that rely on PHP functionality were maintained in the composer.json and provide a node script that referenced composer than moving all of these over to the package.json file.

Reply:
@joemcgill @felixarntz In conjunction with both this and https://github.com/WordPress/performance/pull/1065#discussion_r1546422938, I have added a node script to run PHPUnit tests in https://github.com/WordPress/performance/pull/1065/commits/0ba78d9b6678f672c23d59d442712ae5e3df3f80 and added the scripts in `package.json` due to some NPM specific features.

Now the commands works like:
```
npm run test:php # Run tests for PL plugin
npm run test:php:plugins # Run tests for all plugins
npm run test:php:plugins --plugin=foo # Run tests for foo plugin
npm run test:php:plugins --plugin=foo -- --phpunit-args # Run tests for foo plugin and pass args to PHPUnit.
// ... and same as above for mutlsite just need to append :multisite in commands.
```

Please LMKWYT.

Reply:
One more advantage is, that you can now pass args to PHPUnit by escaping once only like:

```
npm run test:php:plugins --plugin=optimization-detective -- --filter=Foo_Tests
```
is makes it more better IMO.